android/fastlane/Appfile

```diff
@@ -0,0 +1,2 @@
+json_key_file("") # Path to the json secret file - Follow https://docs.fastlane.tools/actions/supply/#setup to get one
+package_name("ch.nevis.mobile.authentication.sdk.react.example") # e.g. com.krausefx.app
```

Comment:
I don't think we need the example in the comment.

Reply:
This is an autogenerated file by Fastlane, I don't really want to touch this...